### ISSUE TITLE
GET /v1/photos/search fails for OAuth Authentication

### ISSUE THREAD

**Issue by lkorth** [500px/api-documentation#161]
Making a request to `GET /v1/photos/search` is successful when a consumer key is included as a query param, however when using an access token and signing the request results in a 401 with the following response

```json
{ 
  "error": "Consumer key missing.",
  "status":401
}
```

According to the [documentation](https://github.com/500px/api-documentation/blob/master/endpoints/photo/GET_photos_search.md), OAuth is a valid form of authentication for this endpoint. Can you confirm is the documentation is wrong or the API is wrong?

**Comment by zimuliu** [comment]
The access token needs to be used together with consumer key to access OAuth API

**Comment by lkorth** [comment]
That's not true, all other endpoints accept _just_ the access token and will return an error when you try to use the access token and consumer key.

**Comment by adrianhartanto0** [comment]
Did you manage to find a solution to this problem?

**Comment by lkorth** [comment]
No, I just switched to using a consumer key for all search requests and have not tried it since.
